packages/perseus/src/widgets/graded-group/__snapshots__/graded-group.test.ts.snap

```diff
@@ -125,7 +125,7 @@ exports[`graded-group should snapshot: initial render (mobile: false) 1`] = `
                                       <div
                                         class="paragraph"
                                       >
-                                         Some bacteria conduct photosynthesis and produce oxygen, much like plants.
+                                        Some bacteria conduct photosynthesis and produce oxygen, much like plants.
```

Comment:
The test data had a leading space. I removed it which caused the need to update snapshots.